ts/features/itwallet/machine/eid/actors.ts

```diff
@@ -154,7 +155,10 @@ export const createEidIssuanceActorsImplementation = (
         walletAttestation: input.walletInstanceAttestation
       });
 
-      trackItwRequest(input.identification.mode);
+      trackItwRequest(
+        input.identification.mode,
+        input.isL3IssuanceEnabled ? "L3" : "L2"
```

Comment:
Here we could be in L2 fallback, so simply checking `isL3IssuanceEnabled` is not correct. Wouldn't be easier to check the `IdentificationContext`?